From add735c0b885cc47cb21a9bca45c4beb00298f27 Mon Sep 17 00:00:00 2001
From: Richard Hansen <rhansen@rhansen.org>
Date: Mon, 4 May 2020 15:58:49 -0400
Subject: [PATCH 35/63] recovery: Set the INFO color to the recovery/fastboot
 light color

The default color clashes with the new recovery logo.

Currently no text is rendered with the INFO color, but that will
change in a future commit.

Change-Id: I737528e0c0a5202532e59d99ca57268478c87965
---
 recovery_ui/screen_ui.cpp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/recovery_ui/screen_ui.cpp b/recovery_ui/screen_ui.cpp
index c2272903..7f80ac95 100644
--- a/recovery_ui/screen_ui.cpp
+++ b/recovery_ui/screen_ui.cpp
@@ -560,7 +560,10 @@ void ScreenRecoveryUI::draw_foreground_locked() {
 void ScreenRecoveryUI::SetColor(UIElement e) const {
   switch (e) {
     case UIElement::INFO:
-      gr_color(249, 194, 0, 255);
+      if (fastbootd_logo_enabled_)
+        gr_color(0xfd, 0xd8, 0x35, 255);
+      else
+        gr_color(0xf8, 0x90, 0xff, 255);
       break;
     case UIElement::HEADER:
       if (fastbootd_logo_enabled_)
-- 
2.34.0

